### PR TITLE
add dbsnp column to mutation table

### PR DESCRIPTION
**Pull request by leexgh**
add dbsnp column to mutation table

image

https://www.ncbi.nlm.nih.gov/snp/rs121913343

### tooltips
- header: 
image

- rs id: 
image

- blank: 
image

### CODE REVIEW

**Comment by leexgh** [comment]
@inodb oh right, I forgot to update it, thanks

**Comment by inodb** [comment]
@leexgh 	Thanks! Could you format the header text a bit nicer so it's easier to read? Maybe something like:

The Single Nucleotide Polymorphism Database (dbSNP) is a free public archive for genetic variation within and across different species.

Although the name of the database implies a collection of one class of polymorphisms only (i.e., single nucleotide polymorphisms (SNPs)), it in fact contains a range of molecular variation:

1. SNPs
2. short deletion and insertion polymorphisms (indels/DIPs)
3. microsatellite markers or short tandem repeats (STRs)
4. multinucleotide polymorphisms (MNPs)
5. heterozygous sequences, and
6. named variants

**Comment by jjgao** [comment]
Thanks, @leexgh. How many our of the 6 categories are we linking to?

File: src/shared/components/mutationTable/MutationTable.tsx [hunk 1]
```diff
@@ -562,6 +565,27 @@ export default class MutationTable<P extends IMutationTableProps> extends React.
             visible: false,
             align: "right"
         };
+
+        this._columns[MutationTableColumnType.DBSNP] = {
+            name: "dbSNP",
+            render: (d:Mutation[]) => DbsnpColumnFormatter.renderFunction(d, this.props.genomeNexusMyVariantInfoCache as GenomeNexusMyVariantInfoCache),
+            sortBy: (d:Mutation[]) => DbsnpColumnFormatter.getSortValue(d, this.props.genomeNexusMyVariantInfoCache as GenomeNexusMyVariantInfoCache),
+            download: (d:Mutation[]) => DbsnpColumnFormatter.download(d, this.props.genomeNexusMyVariantInfoCache as GenomeNexusMyVariantInfoCache),
+            tooltip: (<span style={{maxWidth:370, display:"block", textAlign:"left"}}>
```
Comment (onursumer) [inline src/shared/components/mutationTable/MutationTable.tsx]:
```suggestion
            tooltip: (
                <span style={{maxWidth:370, display:"block", textAlign:"left"}}>
```

File: src/shared/components/mutationTable/MutationTable.tsx [hunk 2]
```diff
@@ -562,6 +565,27 @@ export default class MutationTable<P extends IMutationTableProps> extends React.
...
+                The Single Nucleotide Polymorphism Database (<a href="https://www.ncbi.nlm.nih.gov/snp/" target="_blank">dbSNP</a>) 
+                is a free public archive for genetic variation within and across different species. 
+                <br /><br />Although the name of the database implies a collection of one class of polymorphisms only 
+                (i.e., single nucleotide polymorphisms (SNPs)), it in fact contains a range of molecular variation: 
+                <br /><br />&nbsp;&nbsp;1. SNPs
+                <br />&nbsp;&nbsp;2. short deletion and insertion polymorphisms (indels/DIPs)
+                <br />&nbsp;&nbsp;3. microsatellite markers or short tandem repeats (STRs)
+                <br />&nbsp;&nbsp;4. multinucleotide polymorphisms (MNPs)
+                <br />&nbsp;&nbsp;5. heterozygous sequences
+                <br />&nbsp;&nbsp;6. named variants</span>),
```
Comment (onursumer) [inline src/shared/components/mutationTable/MutationTable.tsx]:
```suggestion
                <br />&nbsp;&nbsp;6. named variants
            </span>
        ),
```

**Comment by leexgh** [comment]
@jjgao are you asking which type do we support?

Cause I get the rs id from myVariant.Info, right now they support insertion, deletion and SNP

**Comment by jjgao** [comment]
Thanks, @leexgh . I was wondering if it's necessary to have such a long tooltip. Maybe we just need the first paragraph? @inodb 

image

**Pull request merged**